job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/additionalTests/JbpmContainerTestMatrix.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.jenkins.jobdsl.templates.additionalTests
+
+import javaposse.jobdsl.dsl.jobs.MatrixJob
+
+class JbpmContainerTestMatrix {
+
+    /**
+     * Adds common configuration to the job.
+     *
+     */
+    static void addDeployConfiguration(MatrixJob matrixJob,
+                                       String kieVersion,
+                                       String jdkVersion,
+                                       String mvnTool,
+                                       String nexusUrl,
+                                       String settingsXml){
+
+        //Add configuration
+        matrixJob.with{
+            def jbpmContainerTest = '''#!/bin/bash -e
```

Comment:
this should be
```
            def jbpmContainerTest = '''#!/bin/bash -e
\n echo "KIE version $kieVersion"
\n echo "Nexus URL:  $nexusUrl"
\n wget -q $nexusUrl/org/jbpm/jbpm/$kieVersion/jbpm-$kieVersion-project-sources.tar.gz -O sources.tar.gz
\n tar xzf sources.tar.gz
\n rm sources.tar.gz
\n mv jbpm-$kieVersion/* .
\n rm -rf jbpm-$kieVersion'''
```
without `'''` content tabulation, or use
`\n rm -rf jbpm-$kieVersion'''.stripMargin()`
as it is done at line 46